src/renderer/system/utils.js

```diff
@@ -44,10 +44,25 @@ export function readClipboard() {
   return clipboard.readText();
 }
 
+export function isUrl(url) {
+  var pattern = new RegExp(
+    '^(https?:\\/\\/)?' +
+      '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.)+[a-z]{2,}|' +
+      '((\\d{1,3}\\.){3}\\d{1,3}))' +
+      '(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*' +
+      '(\\?[;&a-z\\d%_.~+=-]*)?' +
+      '(\\#[-a-z\\d_]*)?$',
+    'i'
```

Comment:
This pattern doesn't appear to support non-ascii characters, and users may store non-ascii URLs in non-punycode format. Imo it would be best to use a combination of [is-url](https://www.npmjs.com/package/is-url) and [is-valid-domain](https://www.npmjs.com/package/is-valid-domain).

If a string passes `is-url` and doesn't have the protocol prefix, ie. `my-domain.com`, pass it to `is-valid-domain` for confirmation that it's a domain and not some random string. This checks the string against valid domain TLDs.

Being that the desktop app is installed and it's already quite large, I don't see the harm in adding these extra packages to allow for this functionality.

Reply:
I had seen some extra package which do these checks, but I thought it was a better idea to implement directly. 

I change this part as you suggest.

Reply:
Fixed.